pkg/lib/cockpit-components-typeahead-select.tsx

```diff
@@ -61,7 +61,7 @@ export interface TypeaheadSelectOption extends Omit<SelectOptionProps, 'content'
 
 export interface TypeaheadSelectProps extends Omit<SelectProps, 'toggle' | 'onSelect'> {
   /** @hide Forwarded ref */
-  innerRef?: React.Ref<any>;
+  innerRef?: React.Ref<HTMLDivElement>;
```

Comment:
Can you please submit this to upstream PF too? 

I get that they may have different indentation rules than us, but I'd rather relax ours for this file than introducing this huge whitespace diff -- it'll make comparing and re-merging unnecessarily hard in the future.

Reply:
Not that replacing "any" here is also due to a lint rule. It's not an actual type error.